src/course6/homework/Challenge7.java

```diff
@@ -0,0 +1,27 @@
+package course6.homework;
+
+import java.util.Scanner;
+
+public class Challenge7 {
+
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+
+        System.out.println("Determining the type of a triangle\n");
+
+        System.out.println("Enter the first side:");
+        double sideOne = scan.nextDouble();
+        System.out.println("Enter the second side:");
+        double sideTwo = scan.nextDouble();
+        System.out.println("Enter the third side:");
+        double sideThree = scan.nextDouble();
+
+        if ((sideOne != sideTwo) && (sideTwo != sideThree) && (sideOne != sideThree)) {
+            System.out.println("The triangle is Scalene.");
+        } else if ((sideOne == sideTwo) && (sideTwo == sideThree )) {
+            System.out.println("The triangle is Equilateral.");
+        } else {
+            System.out.println("The triangle is Isosceles.");
```

Comment:
Reworked. Don't ask me why I did it the way I did it in the first place, because I honestly don't remember. I mainly use the parantheses for clarity, but I'll admit that they can be a problem sometimes.